definitions/infra-mongodb_collection/definition.yml

```diff
@@ -0,0 +1,64 @@
+domain: INFRA
+type: MONGODB_COLLECTION
+goldenTags:
+  - collection
```

Comment:
What is the intent here? There no tag defined as "collection" in the rules
(also not needed since is the entity name, but i'm curious)

Reply:
Makes sense now. I have removed those. Thx!